.chloggen/prw-receiver-use-created-metric.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Should we consider this breaking?  If a user was relying on the series existing, it won't be present anymore.  We might want to consider providing instructions on how to re-enable the created series on prom/prw exporters.

Reply:
> We might want to consider providing instructions on how to re-enable the created series on prom/prw exporters.

This https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17417 should already do the job? It is just about writing it is somewhere?

> If a user was relying on the series existing, it won't be present anymore.

Shall we do something about it or leaving a comment about it in the change log is enough?

Reply:
I think it belongs in _this_ changelog entry.  I.e. "The _created series is removed by the prometheus exporter.  Use foo_bar in the prometheus exporter, or hello_world in the prometheus remote write exporter to re-create the series"

Changelog is enough.

Reply:
Then we probably should also implement exporting the `_created` metric in Prometheus exporter similar to Prometheus remote write exporter before moving forward with this PR?

Reply:
That makes sense to me

Reply:
Ok, will create an issue.

Reply:
@dashpole so here is the issue, it seems that we cannot support `_created` in `exporter/prometheus` now. There is an open issue for supporting _created in `client_golang` https://github.com/prometheus/client_golang/issues/685 that is used in `exporter/prometheus`.

Reply:
Can we put this behind an alpha feature flag then?  Eventually, we will want to always have the behavior added by this PR, but it would break current usage without a way for them to fix it today.

Reply:
Sure, let me add a flag. I'll skip your suggestion related to the changelog entry then.